ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/notification/NotificationResources.java

```diff
@@ -12,66 +12,66 @@
 
 import com.google.gwt.resources.client.ClientBundle;
 import com.google.gwt.resources.client.CssResource;
-import com.google.gwt.resources.client.ImageResource;
 
 import org.vectomatic.dom.svg.ui.SVGResource;
 
-/** @author <a href="mailto:aplotnikov@codenvy.com">Andrey Plotnikov</a> */
+/**
+ * Notifications resources. Contains definition of styles and icons.
+ *
+ * @author Andrey Plotnikov
+ * @author Vlad Zhukovskyi
+ **/
 public interface NotificationResources extends ClientBundle {
-    public interface NotificationCss extends CssResource {
+    interface NotificationCss extends CssResource {
+
         String notificationPanel();
 
-        String notificationPopup();
+        String notificationPanelContainer();
 
-        String notificationItem();
+        String notification();
 
-        String notificationGrid();
+        String notificationIconWrapper();
 
-        String floatLeft();
+        String notificationContentWrapper();
 
-        String margin4px();
+        String notificationTitleWrapper();
 
-        String error();
+        String notificationMessageWrapper();
 
-        String unread();
+        String notificationCloseButtonWrapper();
 
-        String warning();
+        String notificationStatusProgress();
 
-        String success();
+        String notificationStatusSuccess();
 
-        String right25px();
+        String notificationStatusFail();
 
-        String notificationMessage();
+        String notificationPopup();
 
-        String progress();
+        String notificationPopupContentWrapper();
 
-        String countLabel();
+        String notificationPopupIconWrapper();
 
-        String center();
+        String notificationPopupCloseButtonWrapper();
 
-        String close();
+        String notificationPopupTitleWrapper();
 
-        String closePopupIcon();
+        String notificationPopupMessageWrapper();
+
+        String notificationPopupPanel();
+
+        String notificationPopupPlaceholder();
     }
 
     @Source({"notification.css", "org/eclipse/che/ide/api/ui/style.css"})
     NotificationCss notificationCss();
 
-    @Source("message.png")
-    ImageResource message();
-
-    @Source("progress.svg")
-    SVGResource progress();
-
     @Source("success.svg")
     SVGResource success();
 
-    @Source("error.svg")
-    SVGResource error();
+    @Source("fail.svg")
+    SVGResource fail();
 
-    @Source("warning.svg")
-    SVGResource warning();
-
-    @Source("close-popup.svg")
-    SVGResource closePopup();
+    @Source("progress.svg")
+    SVGResource progress();
```

Comment:
Progress, success and error images are used in plugin-openshift. Did you check it?